cloudformation/dynamoDb.yaml

```diff
@@ -220,6 +220,30 @@ Resources:
           - referrals
     DeletionPolicy: Retain
 
+  CampaignReferralsV2DBTable:
```

Comment:
This isn't linked to the api versioning.  It's just that the data structure has changed and I don't want to break the app.  Will remove versioned table names when data is stable and just use original referrals table.